### ISSUE TITLE
Validação flutter

### ISSUE THREAD

**Issue by fpenha** [fpenha/ifeira#2]
Implementação de duas telas, juntamente com alguns possíveis recursos ( botões - tratamento de vento - menu - configuração de perfil) com os seus devidos posicionamentos, a partir da proposta de apresentada pela prototipação. Paralelo a esta implementação desses recursos, a implementação deverá levar em consideração a realização de uma comunicação com o servidor firestore.

**Comment by fpenha** [comment]
Quando estiver estruturando o projeto, leia sobre Bussines Logic Component (BLoC) e o Flutter.

**Comment by marciodouglasIFRN** [comment]
![WhatsApp Image 2019-06-06 at 09 37 27](https://user-images.githubusercontent.com/30964612/59033719-b7d4bb00-883f-11e9-8456-c20da8ab12b5.jpeg)

**Comment by marciodouglasIFRN** [comment]
Fiz essa implementação  para a tela principal do feirante e gostaria de saber se está de acordo com que é esperado

**Comment by fpenha** [comment]
Sim, acredito que estamos chegando ao idealizado. Porém, em sala tínhamos conversado sobre a seguinte ordem:
 - Próxima Feira
 - Pedidos Pendentes 
 - Feiras Agendadas

Independente da ordem, queria conversar contigo sobre a possibilidade de agruparmos **Próxima Feira** com **Pedidos Pendentes**, o intuito dessa mudança é para deixar claro que os **Pedidos Pendentes** é da **Próxima Feira**.

**Comment by fpenha** [comment]
@matheuskaio 

Segue uma proposta, o desafio é trabalharmos as informações necessárias dentro desta sugestão. Perceba que temos o fundo branco, as caixas com um preenchimento de 2.5% de cinza, suas bordas seguem a mesma cor da legenda (confirmada, agendadas). A legenda está substituindo a borda superior. 

No exemplo, a caixa das feiras agendadas não está sendo apresentada, mas ele segue o mesmo padrão descrito. 

![proposta](https://user-images.githubusercontent.com/1133047/59036957-a6db7800-8846-11e9-9c65-ca2cca53bceb.png)

Abraço

Abraço

**Comment by fpenha** [comment]
Proposta evoluída.

Paleta de Cores:
red #EE3239
green #5EAA5F
yellow #FECE00
orange #ff6f0f
grape #9D6AB9

![proposta](https://user-images.githubusercontent.com/1133047/59041912-e9ee1900-884f-11e9-93f7-307c28882a11.png)

![proposta-orange](https://user-images.githubusercontent.com/1133047/59043121-0d19c800-8852-11e9-95b3-b3ff856cdd5f.png)

Eu gostei mais do Laranja, na partes das Feiras Agendadas

**Comment by matheuskaio** [comment]
@fpenha estamos trabalhando nessa sua proposta.
